sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorKeyCredential.cs

```diff
@@ -46,26 +46,26 @@ internal string ApiKey
         /// Updates the service key. This is intended to be used when you've regenerated
         /// your service key and want to update long lived clients.
         /// </summary>
-        /// <param name="key">Key to authenticate the service against.</param>
-        /// <exception cref="ArgumentNullException"><paramref name="key"/> is null.</exception>
-        /// <exception cref="ArgumentException"><paramref name="key"/> is empty.</exception>
-        public void UpdateSubscriptionKey(string key)
+        /// <param name="subscriptionKey">Key to authenticate the service against.</param>
```

Comment:
do you think the docstring should say `subscription key` ?

Reply:
Updated! I see no harm in being more specific.